src/full/Agda/Interaction/Highlighting/Precise.hs

```diff
@@ -301,7 +301,7 @@ compressedFileInvariant :: CompressedFile -> Bool
 compressedFileInvariant (CompressedFile []) = True
 compressedFileInvariant (CompressedFile f)  =
   all rangeInvariant rs &&
-  all (not . empty) rs &&
+  not (any empty rs) &&
```

Comment:
Breaks the conjunctive pattern `all ... && all ... && and` (these are all conjunctions).